### PR TITLE
Support for Server Tokens

### LINKED ISSUE #155: Add Support for SRCDS_TOKEN to Appear on the Sever List
Add support for a SRCDS_TOKEN environment variable to pass in when running the game in order for the server to appear on the server browser.

### PR DESCRIPTION
**Pull request by joshuafhiggins**
Users would be able to specify a game server token from Steam under SERVER_TOKEN in order to appear on the server browser list. Fixes #155.

### CODE REVIEW

File: packages/tf2-base/Dockerfile [hunk 1]
```diff
@@ -67,6 +67,7 @@ ENV PORT=27015
 ENV CLIENT_PORT=27016
 ENV STEAM_PORT=27018
 ENV STV_PORT=27020
+ENV SEVER_TOKEN=""
```
Comment (garrappachc) [inline packages/tf2-base/Dockerfile:70]: Typo: `SERVER_TOKEN`

**Review by garrappachc — CHANGES_REQUESTED** [review]

File: packages/tf2-base/Dockerfile (hunk 1 above)
Comment (joshuafhiggins) [reply packages/tf2-base/Dockerfile]: Fixed!

**Review by garrappachc — APPROVED** [review]

**Comment by garrappachc** [comment]
Thanks!

**Comment by joshuafhiggins** [comment]
You're welcome! On another note, would there be any interest in including some other environmental controls in tf2-base, like random crits, fixed shot patterns, and server tags? I think these would be very valuable for people to easily setup a server on their own with settings that are commonly changed. I'm willing to submit a PR for them, but obviously have the default values for them be what the default is now for the image as to not switch settings out from under people.

**Comment by garrappachc** [comment]
> You're welcome! On another note, would there be any interest in including some other environmental controls in tf2-base, like random crits, fixed shot patterns, and server tags? I think these would be very valuable for people to easily setup a server on their own with settings that are commonly changed. I'm willing to submit a PR for them, but obviously have the default values for them be what the default is now for the image as to not switch settings out from under people.

These settings are a part of the `server.cfg` file, so they're easily accessible already; all one has to do is to provide their own `server.cfg.template` and reference env variables in it. Unless mounting the `server.cfg.template` is an issue, I don't see any value in doing this.

**Comment by joshuafhiggins** [comment]
> These settings are a part of the `server.cfg` file, so they're easily accessible already; all one has to do is to provide their own `server.cfg.template` and reference env variables in it. Unless mounting the `server.cfg.template` is an issue, I don't see any value in doing this.

Ah I see, I didn't know you could mount a custom server.cfg.template, my bad!

**Pull request merged**